backend/apps/accounts/views.py

```
@@ -109,17 +100,60 @@ class SignupView(generics.CreateAPIView):
     throttle_classes = [SignupThrottle]
 
 
+@extend_schema(
+    summary="Register a new user",
+    description="Create a new user account with username, email, and password",
+    request=UserCreateSchema,
```

Ruff (0.15.21)

[error] Undefined name `UserCreateSchema`

(F821)